sys-fs/squashfs_tools/squashfs_tools-4.3.recipe

```diff
@@ -0,0 +1,78 @@
+SUMMARY="Provides tools for creating and manipulating squashfs filesystems"
+DESCRIPTION="squashfs-tools provides tools for creating and appending to \
+(mksquashfs) and extracting (unsquashfs) squashfs filesystems.
+
+Squashfs is a compressed read-only filesystem. It is intended for general \
+read-only filesystem use, archival use (i.e. in cases where a .tar.gz file \
+may be used, and in constrained block device/memory systems (e.g. embedded \
+systems) where low overhead is needed."
+HOMEPAGE="http://squashfs.sourceforge.net"
+COPYRIGHT="2002-2014 Philip Lougher"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://prdownloads.sf.net/squashfs/squashfs$portVersion.tar.gz"
```

Comment:
I think we should avoid the old "prdownloads.sf.net" name and, instead, use "downloads.sf.net" or "downloads.sourceforge.net":
```
SOURCE_URI="https://downloads.sf.net/squashfs/squashfs$portVersion.tar.gz"
```